voicevox_engine/model.py

```diff
@@ -104,68 +106,11 @@ def __init__(self, err: ParseKanaError):
         super().__init__(text=err.text, error_name=err.errname, error_args=err.kwargs)
 
 
-class SpeakerStyle(BaseModel):
-    """
-    スピーカーのスタイル情報
-    """
-
-    name: str = Field(title="スタイル名")
-    id: int = Field(title="スタイルID")
-
-
-class SpeakerSupportPermittedSynthesisMorphing(str, Enum):
-    ALL = "ALL"  # 全て許可
-    SELF_ONLY = "SELF_ONLY"  # 同じ話者内でのみ許可
-    NOTHING = "NOTHING"  # 全て禁止
-
-    @classmethod
-    def _missing_(cls, value: object) -> "SpeakerSupportPermittedSynthesisMorphing":
-        return SpeakerSupportPermittedSynthesisMorphing.ALL
-
-
-class SpeakerSupportedFeatures(BaseModel):
-    """
-    話者の対応機能の情報
-    """
-
-    permitted_synthesis_morphing: Optional[
-        SpeakerSupportPermittedSynthesisMorphing
-    ] = Field(
-        title="モーフィング機能への対応", default=SpeakerSupportPermittedSynthesisMorphing(None)
-    )
-
-
-class Speaker(BaseModel):
-    """
-    スピーカー情報
-    """
-
-    name: str = Field(title="名前")
-    speaker_uuid: str = Field(title="スピーカーのUUID")
-    supported_features: Optional[SpeakerSupportedFeatures] = Field(title="スピーカーの対応機能")
-    styles: List[SpeakerStyle] = Field(title="スピーカースタイルの一覧")
-    version: str = Field("スピーカーのバージョン")
-
-
-class StyleInfo(BaseModel):
-    """
-    スタイルの追加情報
-    """
-
-    id: int = Field(title="スタイルID")
-    icon: str = Field(title="当該スタイルのアイコンをbase64エンコードしたもの")
-    portrait: Optional[str] = Field(title="当該スタイルのportrait.pngをbase64エンコードしたもの")
-    voice_samples: List[str] = Field(title="voice_sampleのwavファイルをbase64エンコードしたもの")
-
-
-class SpeakerInfo(BaseModel):
-    """
-    話者の追加情報
-    """
+class MorphableTargetInfo(BaseModel):
 
-    policy: str = Field(title="policy.md")
-    portrait: str = Field(title="portrait.pngをbase64エンコードしたもの")
-    style_infos: List[StyleInfo] = Field(title="スタイルの追加情報")
+    is_morphable: bool = Field(title="指定したbaseに対してモーフィングの可否")
```

Comment:
英語の変数名などを使うと意図が曖昧になるのでちゃんと翻訳したほうが良いかもとちょっと思いました！
```suggestion
    is_morphable: bool = Field(title="指定した話者に対してモーフィングの可否")
```